rclcpp_examples/CMakeLists.txt

```diff
@@ -103,8 +103,18 @@ if(AMENT_ENABLE_TESTING)
 
   # Add each test case.  Multi-executable tests can be specified in
   # semicolon-separated strings, like  exe1;exe2.
-  foreach(example_executable list_parameters_async list_parameters parameter_events_async parameter_events set_and_get_parameters_async set_and_get_parameters "talker;listener")
-
+  set(example_executables
+    list_parameters_async
+    list_parameters
+    parameter_events_async
+    parameter_events
+    set_and_get_parameters_async
+    set_and_get_parameters
+    "talker\;listener"
+    "add_two_ints_client\;add_two_ints_server"
+    "add_two_ints_client_async\;add_two_ints_server")
```

Comment:
While using `\;` here works it is very fragile. E.g. assigning the variable `example_executables` to `example_executables2` would already break it. Therefore I would suggest to use a different separate (e.g. `:`) and replace it within the first loop with a `semicolon`.